pages/docs/reference/functions.md

```diff
@@ -53,6 +53,20 @@ fun powerOf(number: Int, exponent: Int) { /*...*/ }
 ```
 </div>
 
+The Kotlin code style recommends using a [trailing comma](coding-conventions.html#trailing-commas) when you declare function parameters.
+It simplifies reordering, deleting, and adding new parameters. Using trailing commas are optional: without them your code will work as well.
```

Comment:
Do we really need to repeat this on three pages? "Optional" is obvious from the example above and the `recommends` word, benefits are described in coding conventions. The same goes for the Classes page.

Reply:
IIRC, you've mentioned above that you want to see here the benefits of using trailing commas ("when is this comma useful?"). Did I miss something?